src/predict.rs

```diff
@@ -183,6 +184,21 @@ impl PredictionMode {
       || self == PredictionMode::NEW_NEAR1MV
       || self == PredictionMode::NEW_NEAR2MV
   }
+  pub fn ref_mv_idx(self) -> usize {
```

Comment:
For consistency with LTO-enabled builds and the neighbouring methods of similar complexity, we may add `#[inline]` here.